target/bin/getmail-cron

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+for file in /etc/getmailrc.d/getmailrc*; do
+  if ! pgrep -f "${file}"$ > /dev/null 2>&1; then
+    /usr/bin/getmail --getmaildir /var/lib/getmail --rcfile "${file}"
+  fi
+done
```

Comment:
```suggestion
#! /bin/bash

for FILE in /etc/getmailrc.d/getmailrc*
do
  if ! pgrep -f "${FILE}"$ > /dev/null 2>&1
  then
    /usr/bin/getmail --getmaildir /var/lib/getmail --rcfile "${FILE}"
  fi
done
```

Is the `$` after `"${FILE}"` intended?

Reply:
it was intended, to make sure it was matching at the end of the command.  Not sure why I did that though  ... could probably take that out